go-controller/pkg/node/healthcheck.go

```diff
@@ -138,6 +139,32 @@ func checkDefaultConntrackRules(gwBridge string, nFlows int, stopChan chan struc
 					nFlows, out)
 				os.Exit(1)
 			}
+
+			// it could be that the ovn-controller recreated the patch between the host OVS bridge and
+			// the integration bridge, as a result the ofport number changed for that patch interface
+			curOfportPatch, stderr, err := util.RunOVSVsctl("--if-exists", "get", "Interface", patchIntf, "ofport")
+			if err != nil {
+				klog.Errorf("Failed to get ofport of %s, stderr: %q, error: %v", patchIntf, stderr, err)
+				continue
+			}
+			if ofportPatch != curOfportPatch {
+				klog.Errorf("fatal error: ofport of %s has changed from %s to %s",
+					patchIntf, ofportPatch, curOfportPatch)
+				os.Exit(1)
```

Comment:
why not reprogram the flows instead of exiting?

Reply:
@trozet we will also have to re-program all the NodePort services on the switch. The NodePort services will be using the old patch port ofproto number. We will be addressing, in an another PR, how to reinstate all the flows without having to kill the ovnkube-node daemon. For now, this PR addresses the immediate issue we are seeing in our clusters.